src/bentoml/_internal/service/openapi/utils.py

```diff
@@ -43,6 +52,33 @@ def pydantic_components_schema(pydantic_model: t.Type[pydantic.BaseModel]):
     return {k: Schema(**definitions[k]) for k in sorted(definitions)}
 
 
+def typeddict_components_schema(typeddict: t.Type[TypedDict]):
```

Comment:
```suggestion
def typeddict_components_schema(typeddict: type[TypedDict]):
```

as much as possible all new code should follow PEP604